gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationTest.java

```diff
@@ -76,9 +76,10 @@
           put("fs.gs.glob.algorithm", GlobAlgorithm.CONCURRENT);
           put("fs.gs.grpc.checksums.enable", false);
           put("fs.gs.grpc.enable", false);
-          put("fs.gs.grpc.server.address", null);
           put("fs.gs.grpc.read.timeout.ms", 20 * 60 * 1000L);
           put("fs.gs.grpc.read.metadata.timeout.ms", 60 * 1000L);
+          put("fs.gs.grpc.server.address", null);
```

Comment:
Why not use `GCS_GRPC_SERVER_ADDRESS`?

Reply:
The `DEFAULT_GCS_GRPC_SERVER_ADDRESS` is retrieved from the client, Is the idea to use `GCS_GRPC_SERVER_ADDRESS` string as the value ?

Since the default [configuration](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java#L355) is null, it makes sense to maintain the same